include/aws/crt/auth/Credentials.h

```diff
@@ -448,6 +560,20 @@ namespace Aws
                     const CredentialsProviderDelegateConfig &config,
                     Allocator *allocator = ApiAllocator());
 
+                /**
+                 * Creates a provider that sources credentials from the Cognito Identity service
```

Comment:
Super minor nitpick: Should `Cognito Identity service` be `Cognito credentials provider`? Or are they separate things?

Reply:
The Cognito-Identity is the web service that the provider sources from, so it should be good as written.